htdocs/beta/helper/common.php

```diff
@@ -44,6 +46,26 @@ function button($path, $caption, $icon, $background_color, $link = true) {
     }
   }
 
+  function button_small($path, $caption, $icon, $background_color, $link = true) {
```

Comment:
On peut, je sais que ca serait plus correct, mais ca veut dire qu'il faut changer toutes les occurrences de button() ? 
Je n'ai pas envie de faire ça maintenant, mais is ont peut le faire sans ok.